azure-pipelines.yml

```diff
@@ -37,7 +37,7 @@ stages:
   variables:
   - template: .azure-pipelines/azure-pipelines-repd-build-variables.yml
   - name: CACHE_MODE
-    value: rcache
+    value: none
```

Comment:
why change this?

Reply:
Use this to identify a potential issue. which addressed at https://github.com/Azure/sonic-buildimage/pull/9705#pullrequestreview-847106456
this will be removed.
Thanks for the question and sorry for the confusion :)